style.css

```diff
@@ -484,7 +488,363 @@ footer {
 .liner {
   height: 5px;
   width: 134px;
-  margin: 13px 121px 0 120px;
+  margin-top: 13px;
+  margin-left: 125px;
   background-color: #0d151e;
   border-radius: 10px;
 }
+
+/* -------------------------------------------------------
+ -------desktop version style using media query--------
+------------------------------------------------- */
+
+@media screen and (min-width: 768px) {
+  body {
+    width: 100%;
+  }
+
+  /* header/ navbar media query start */
+
+  .navbar-container {
+    min-height: 72px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
```

Comment:
- **[OPTIONAL]** The objective of this exercise is to match the Figma design, please 🙏🏽 adjust the navbar height and the mail button size to be similar to the Figma template.

| Figma template | Your design |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/214885724-ff3ae483-086d-4b00-93eb-2b2543b7b7be.png) | ![image](https://user-images.githubusercontent.com/58642949/214885853-ac0b0afc-c297-4f05-9331-8ffe68b88e9b.png) |